specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2021-11-01-preview/managedClusters.json

```diff
@@ -519,6 +519,9 @@
           "Create/Update AAD Managed Cluster with EnableAzureRBAC": {
             "$ref": "./examples/ManagedClustersCreate_UpdateWithEnableAzureRBAC.json"
           },
+          "Create/Update AAD Managed Cluster with EnableNamespaceResources": {
```

Comment:
this gives perception that its supported only with AAD managed cluster. Let's remove this as namespace feature is supported for non AAD cluster too.

Reply:
done